test/modules/TwinTester/TwinTesterUtil.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TwinTester
+{
+    using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Shared;
+
+    static class TwinTesterUtil
+    {
+        public static TwinCollection GetResetedReportedPropertiesTwin(Twin originalTwin)
```

Comment:
currently have 2 usages; and both will use this method output to call moduleClient to reset report properties.  I think it be better to make this method to take moduleClient and do cleanup here.  what do you think?